iOS/XCharts.iOS/XCharts.iOS.csproj

```diff
@@ -1,160 +1,18 @@
-    <Compile Include="Models\Styles\BaseStyle.cs" />
-    <Compile Include="Helpers\DrawHelper.cs" />
-    <Compile Include="Models\Styles\Position.cs" />
-    <Compile Include="Extensions\CGExtensions.cs" />
-    <Compile Include="Abstract\Renders\IAxisRender.cs" />
-    <Compile Include="Abstract\Renders\IXAxisRender.cs" />
-    <Compile Include="Abstract\Renders\IYAxisRender.cs" />
-    <Compile Include="Implementation\Renders\XAxisRender.cs" />
-    <Compile Include="Models\Renders\Axis\AxisStyle.cs" />
-    <Compile Include="Models\Renders\Axis\XAxisStyle.cs" />
-    <Compile Include="Models\Renders\Axis\XAxisPosition.cs" />
-    <Compile Include="Models\Renders\Axis\YAxisStyle.cs" />
-    <Compile Include="Implementation\Renders\YAxisRender.cs" />
-    <Compile Include="Abstract\Handlers\Tap\IPointTapHandler.cs" />
-    <Compile Include="Implementation\Handlers\Tap\PointTapHandler.cs" />
-    <Compile Include="Abstract\Handlers\Highlight\IHighlightHandler.cs" />
-    <Compile Include="Implementation\Axises\BaseAxis.cs" />
-    <Compile Include="Views\Chart\ChartView.cs" />
-    <Compile Include="Implementation\DataSources\PointEntryChartDataSource.cs" />
-    <Compile Include="Implementation\Entries\EntryPointWrapper.cs" />
-    <Compile Include="Abstract\Managers\IManager.cs" />
-    <Compile Include="Implementation\Manager\DescreteManager.cs" />
-    <Compile Include="Abstract\Renders\IManualRender.cs" />
-    <Compile Include="Implementation\Renders\BaseRender.cs" />
-    <Compile Include="Implementation\Manager\BaseManager.cs" />
-    <Compile Include="Models\Renders\Axis\YAxisPosition.cs" />
-    <Compile Include="Models\Styles\GridStyle.cs" />
-    <Compile Include="Abstract\Handlers\Highlight\IPointHighlightHandler.cs" />
-    <Compile Include="Abstract\Axises\IAxisRangeProvider.cs" />
-    <Compile Include="Implementation\Axises\EntriesCountAxisRangeProvider.cs" />
-    <Compile Include="Implementation\Axises\MultiplierAxisRangeProvider.cs" />
-    <Compile Include="Implementation\Axises\OffsetAxisRangeProvider.cs" />
-    <Compile Include="Implementation\Axises\PointRangeProvider.cs" />
-    <Compile Include="Abstract\Axises\IAxisValuesPositionsProvider.cs" />
-    <Compile Include="Implementation\Axises\StepAxisValuesPositionsProvider.cs" />
-    <Compile Include="Implementation\Axises\CountAxisValuesPositionsProvider.cs" />
-    <Compile Include="Implementation\Handlers\Tap\BaseTapHandler.cs" />
-    <Compile Include="Abstract\Handlers\Tap\ITapHandler.cs" />
-  </ItemGroup>
-  <Import Project="$(MSBuildExtensionsPath)\Xamarin\iOS\Xamarin.iOS.CSharp.targets" />
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFramework>net6.0-ios</TargetFramework>
+        <OutputType>Library</OutputType>
+        <Nullable>disable</Nullable>
+        <ImplicitUsings>true</ImplicitUsings>
+
+
+        <SupportedOSPlatformVersion>13.0</SupportedOSPlatformVersion>
+        <ApplicationId>emportal</ApplicationId>

```

Comment:
The name of the package is `XCharts.iOS`. Shouldn't we use the `PackageId` here instead of the `ApplicationId`?